CompilerSource/Makefile

```diff
@@ -66,6 +67,9 @@ clean:
 $(TARGET): $(OBJECTS)
 	$(CXX) $(LDFLAGS) -o $@ $(OBJECTS) $(LDLIBS)
 
+emake:
```

Comment:
we don't need a rule to build emake in the compiler as theres one in the root makefile